bazel/foreign_cc/BUILD

```diff
@@ -157,7 +157,7 @@ configure_make(
         "--disable-manual",
         "--disable-shared",
         "--enable-static",
-        "--with-colm=$EXT_BUILD_DEPS/colm",
+        "--with-colm=$$EXT_BUILD_DEPS/colm",
```

Comment:
not sure what has changed necessitating this - or tbh how it worked previously but it appears to resolve